pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
```

Comment:
I'm reluctant to removing it due to comments "we need to support it for backward compatibility" (cc @wojtek-t). I believe it's ok to remove these labels during a transition to newer api version. I'd rather not touch it now when `v1` is stabilizing.

Added deprecation warnings in comments.